test/runner.py

```diff
@@ -284,7 +284,7 @@ def load_test_suites(args, modules):
       for test in tests:
         suite.addTest(test)
       suites.append((m.__name__, suite))
-  if total_tests == 1:
+  if total_tests == 1 or parallel_testsuite.num_cores() == 1:
```

Comment:
Is this related?

Reply:
Yes, when running 4gb tests I run with `--cores=1` in which case I want use save the test results in `out/test` .. just like if we run a single test